lib/floe/workflow/states/task.rb

```diff
@@ -1,11 +1,16 @@
 # frozen_string_literal: true
 
+require_relative "input_output_mixin"
+require_relative "non_terminal_mixin"
+require_relative "retry_catch_mixin"
```

Comment:
Feels like these belong with the other `requires` in the global `floe.rb`

Reply:
Yes, it is there as well but keeping those alphabetic means this fails to resolve the constant.
Alternatively, I can move all of the _mixin requires above the others in floe.rb

Reply:
I like making the require not alphabetical.

Reply:
I like the require being in the place that needs it and not trying to solve a dependency graph in floe.rb :laughing:

Reply:
huh.

We have been putting all requires up front in `floe.rb`.
(I did not just say: we have been doing it this way and we have to continue doing that)

Do we want to get away from that?

wondering if InputOutputMixin and the others do not belong in States.
That way we can just require these and then all states and not have to deal with orders.

Does look like these 3 mixins are used by only states

Reply:
> We have been putting all requires up front in floe.rb.
> (I did not just say: we have been doing it this way and we have to continue doing that)

No I don't think we need to stop doing that, I think of `require 'floe'` as "just require everything" but if a particular class needs something specific it should require it that way a caller could `require 'floe/workflow/context'` and not having to bring in the memory of the entire gem.